.xo-config.json

```diff
@@ -47,7 +47,7 @@
 	"overrides": [
 		{
 			"files": ["sdk.mjs", "sdk.d.ts"],
-			"nodeVersion": ">=14.13.1",
+			"nodeVersion": "^20.11 || >=22",
```

Comment:
Just letting a comment here for further reviews...

The version `20.11` added `import.meta.dirname` and `import.meta.filename` (see https://nodejs.org/es/blog/release/v20.11.0).

Reply:
I'm unable to add a comment since it's a JSON file. But we can add the comment after the #13015.